demo/tutorial_notebook/animal_story.ipynb

```diff
@@ -59,12 +59,12 @@
     "# MidJourney demo\n",
     "\n",
     "```\n",
-    "custom_function: demo.logo_generation.logo_generation\n",
+    "custom_function: demo.animal_story.image_generation\n",
     "description: Generated experiment config\n",
     "dataset:\n",
     "  data_generators:\n",
     "    openai_prompt_data_generator:\n",
-    "      chunk_size: 100000\n",
+    "      chunk_size: 2\n",
```

Comment:
Leave the chunk_size = 10000 , it doesn't matter

Reply:
solved